.github/integration/tests/common/30_ingest_test.sh

```diff
@@ -123,5 +122,40 @@ for file in dummy_data.c4gh largefile.c4gh; do
        sleep 10
     done
     
+    dataset=$(printf "EGAD%011d" "$count" )
+
+   # Map dataset ids
+   curl -vvv -u test:test 'localhost:15672/api/exchanges/test/sda/publish' \
+   -H 'Content-Type: application/json;charset=UTF-8' \
+   --data-binary "$( echo '{
+                           "vhost":"test",
+                           "name":"sda",
+                           "properties":{
+                              "delivery_mode":2,
+                              "correlation_id":"CORRID",
+                              "content_encoding":"UTF-8",
+                              "content_type":"application/json"
+                           },
+                           "routing_key":"files",
+                           "payload_encoding":"string",
+                           "payload":"{
+                              \"type\":\"mapping\",
+                              \"dataset_id\":\"DATASET\",
+                              \"accession_ids\":[\"ACCESSIONID\"]}"
+                           }'| sed -e "s/DATASET/$dataset/" -e "s/ACCESSIONID/$access/" -e "s/CORRID/$count/")"
+
+   dbcheck=$(docker run --rm --name client --network dev_utils_default \
+   neicnordic/pg-client:latest postgresql://lega_out:lega_out@db:5432/lega \
+   -c "SELECT * from local_ega_ebi.file_dataset where dataset_id='$dataset' and file_id='$access'")
```

Comment:
This will have output with column headers and thus always have a length, a `-t` should fix that.

Reply:
When you get a response yes, if there is no match to the query you get a null response.